web-components/src/mixins/FocusTrapMixin.ts

```diff
@@ -179,9 +179,10 @@ export const FocusTrapMixin = <T extends AnyConstructor<FocusClass & FocusTrapCl
 
     private shouldSkipFocus(element: HTMLElement) {
       // when combobox is having more than 100 items screen getting freezed
-      if(element.className.split(" ").indexOf('md-combobox')>-1){
-        return true;
+      if(element.className.split && element.className.split(" ").indexOf('md-combobox')>-1){
```

Comment:
what is a reason split function is missing? className can also be empty.

Reply:
For the svg element, .className is giving object instead of string, so split function is missing. Also doing the changes to handle for className if missing.